client/dist/styles/questionsAnswers.css

```diff
@@ -308,7 +227,119 @@ p.answerHelpfulText, p.userHelpfulBtn, p.userReportBtn{
   margin-right: 2.5em;
   justify-content: start;
 }
+
+/* questionAdd form CSS */
+.addQuestionAsterisk.productTitle, .addQuestionAsterisk.userNickName, .addQuestionAsterisk.userEmail {
+  margin: 1vh;
+}
+.addQuestionAsterisk.productTitle:after, .addQuestionAsterisk.userNickName:after, .addQuestionAsterisk.userEmail:after{
+  content: ' *';
+  color: #B22222;
+  height: 3vh;
+}
+.qFormData {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  background-color: lightsteelblue;
```

Comment:
This is a new color to me, look forward to seeing it. If it looks good, I'll probably involve it in my component so we get a more cohesive theme going on